scripts/examples/01-Camera/03-Event-Cameras/02-Genx320/genx320_event_mode.py

```diff
@@ -0,0 +1,56 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2025 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# This example shows off using the genx320 event camera from Prophesee
+# using event streaming mode.
+
+import csi
+import image
+import time
+# https://micropython-ulab.readthedocs.io/en/latest/index.html
+from ulab import numpy as np
+
+# Surface to draw the histogram image on.
+img = image.Image(320, 320, image.GRAYSCALE)
+
+# ndarray to hold events from the camera
+# must be EVT_res events by 6 values
+#
+# 0: event type
+# 1: seconds timestamp
+# 2: milliseconds timestamp
+# 3: microseconds timestamp
+# 4: x coordinate (0-319 for the genx320)
+# 5: y coordinate (0-319 for the genx320)
+events = np.zeros((2048, 6), dtype=np.uint16)
+
+# Initialize the sensor.
+csi0 = csi.CSI(cid=csi.GENX320)
```

Comment:
```suggestion
csi0 = csi.CSI()
```
You only have to specify the ID if it's used with another sensor, otherwise you get the default and only sensor.

Reply:
Yes, my thought here is that by specifying the cid, the script will work for the upcoming dual module too. Is that sensible?

Reply:
Yes, good point. Though it will probably also require its own example(s).